### ISSUE TITLE
Firefox: Button selection

### ISSUE THREAD

**Issue by PaulWoitaschek** [mdbootstrap/mdb-ui-kit#792]
When using firefox every button I click gets selected and highlighted which looks ugly. Is it possible to disable this?

<img width="1440" alt="screen shot 2015-12-24 at 13 50 06" src="https://cloud.githubusercontent.com/assets/743462/11994810/5c3c4744-aa45-11e5-951d-7440f88974ed.png">

**Comment by rosskevin** [comment]
Yes,  I'll check the autoprefixer in an hour our so.  It needs to be generating something like ‘-moz-inner-outline:  none‘

**Comment by rosskevin** [comment]
This is in our version:

``` css
.btn::-moz-focus-inner,
.input-group-btn .btn::-moz-focus-inner {
  border: 0;
}
```

This is what I have generated in v4 (which is from Bootstrap's normalize)

``` css
/* line 315, /Users/kross/alienfast/bootstrap-material-design/bower_components/bootstrap/scss/_normalize.scss */
button::-moz-focus-inner,
input::-moz-focus-inner {
  padding: 0;
  border: 0;
}
```

**Comment by rosskevin** [comment]
https://css-tricks.com/removing-the-dotted-outline/

**Comment by rosskevin** [comment]
Confirmed, fixed in 0.5.7
